cpp/min_path_to_Bucharest.cpp

```diff
@@ -60,6 +61,11 @@ bool PathSearchNode::IsSameState( PathSearchNode &rhs )
   return(false);
 }
 
+size_t PathSearchNode::Hash()
+{
+  return hash<int>{}(city);

```

Comment:
City is an enum without a type specified. So, it would be an integral type that can handle all the values specified. Since that is not necessarily fixed, it may not be int (could be optimized to an int8_t or something else). I'd suggest specifying the storage type definition of the enum so it matches this hash template type.

Reply:
Done